code/modules/mining/mint.dm

```diff
@@ -132,7 +142,7 @@
 					coinsToProduce--
 					newCoins++
 					src.updateUsrDialog()
-					sleep(5);
```

Comment:
Holy fucking shitfuck these had colons to end the line, I don't know who though that the sleeps in specific needed to go against the style of pretty much everything else.